src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCount.java

```diff
@@ -14,6 +15,7 @@
 public final class AllelicCount implements Locatable {
     private final SimpleInterval interval;
     private final int refReadCount, altReadCount;
+    private Nucleotide refNucleotide, altNucleotide;
```

Comment:
Ah I see. what you need to do is make the most-parameter rich constructor the one that actually has the most code; then the constructor that does not take values for those two fields passes nulls to the other constructor.